apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html

```diff
@@ -8,7 +8,7 @@
       (fileClick)="fileClick($event)"
     >
       <sky-file-attachment-label>
-        Attach any file
+        Birth certificate
```

Comment:
can we update the code example too? Attach any file doesn't make much sense with the new labels

Reply:
Good catch - updated.